pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -18,10 +20,6 @@ stdenv.mkDerivation rec {
     chmod -x $dest/*.ttf
   '';
 
-  outputHashAlgo = "sha256";
-  outputHashMode = "recursive";
-  outputHash = "0q03gg0sh2mljlbmhamnxz28d13znh9dzca84p554s7pwg6z4wca";
```

Comment:
The remaining duplicates are:

* ./ofl/alefhebrew/Alef-Bold.ttf and ./ofl/alef/Alef-Bold.ttf
* ./ofl/alefhebrew/Alef-Regular.ttf and ./ofl/alef/Alef-Regular.ttf
* ./ofl/mrbedfort/MrBedfort-Regular.ttf and ./ofl/mrbedford/MrBedfort-Regular.ttf
* ./ofl/siemreap/Siemreap.ttf and ./ofl/siamreap/Siemreap.ttf
* ./ofl/terminaldosislight/TerminalDosis-Light.ttf and ./ofl/terminaldosis/TerminalDosis-Light.ttf